text/0035-verifiable-burning.md

```diff
@@ -0,0 +1,165 @@
+this data alone can be used to spend the funds, that indicates a design flaw in the transaction protocol. However,
+it is conceivable (in fact, likely) that for *almost all* view keys, the view key is not enough for the attacker
+to figure out how to spend the funds, but for a small number of pathological view keys, it is enough.
+In the rest of this section we will demonstrate that the burn address we picked is not pathological in this way.
+
+In order to spend the burned funds, an attacker needs to be able to obtain the subaddress spend private key corresponding
+to the burn address. Therefore, our goal is to show that it is infeasible to learn that subaddress spend private key.
+(This condition also implies that the account spend private key is infeasible to learn, since that key could be used
+to derive the subaddress spend private key.)
+
+The subaddress spend private key is the root of the subaddress spend public key, relative to the Ristretto basepoint.
+So this means, solving the discrete logarithm problem for the subaddress spend public key, given the additional information
+available about the burn address.
+
+The security argument has two steps:
+* First, we claim that the subaddress spend public key is pseudorandom. If there is an algorithm which, given the burn address
+  and burn address view private key, produces the spend private key with high probability, then this algorithm also works with
+  high probability when the construction is modified so that the spend public key is chosen uniformly at random.
+  
+  This claim can be justified by thinking of `Blake2b` as a random oracle. (There are other less powerful assumptions that we
+  could make here, but in the interest of brevity we will omit discussion.)
+  
+  Another way to think about it is that we are claiming
+  that the spend public key we constructed is a ``nothing up my sleeve number'' and there is no way that it could have been
+  manipulated to have non-random properties. (This assumption is also used to construct Pedersen generators for MobileCoin amount commitments.)
+* Second, if there is an algorithm `A` which given a random curve point, the scalar `[1u8; 32]`, and the product of that random
+  curve point and that scalar, finds the discrete log of the curve point with high probability, then this is also an algorithm for solving
+  discrete log, since for any curve point, we can efficiently multiply it by `[1u8; 32]`, and then we would be able to use
+  `A` as a black box to solve discrete log with high probability for random inputs. This directly contradicts the hardness assumption
+  for discrete log in curve25519.
+
+Thus, we have shown by reduction to discrete log that it is intractable to find the subaddress spend key for these funds.
+
+This security argument also shows why it isn't important for the view private key to be chosen by hashing.
+
+# Prior art
+[prior-art]: #prior-art
+
+None at this time.
```

Comment:
You could link to the previous proof-of-concept implementations [here](https://github.com/christian-oudard/mc_incinerator) and [here](https://github.com/mobilecoinfoundation/mobilecoin/pull/1581)?